src/components/WalletCongresSenate/Motion/index.js

```diff
@@ -72,15 +43,49 @@ export default function Motion({
     };
     dispatch(voteMotion(voteMotionData));
   };
+
   return (
-    <div className={stylesPage.stakingWrapper}>
-      <Card className={stylesPage.overviewWrapper}>
-        <div className={styles.metaInfoLine}>
-          <p>
-            Proposal id:
-            <b>{truncate(proposal, 13)}</b>
-          </p>
-          <span>
+    <Card
+      title={(
+        <>
+          Proposal id:
+          <b>{truncate(proposal, 13)}</b>
+        </>
+      )}
+      className={styles.fullWidth}
+      actions={userIsMember ? [
+        isClosable && (
+          <Button
```

Comment:
These buttons dont show probably because userIsMember is somehow borked

Reply:
Unborked